changelog.json

```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "2.4.1-beta1": [
+      "[Added] Indicate which commits are un-pushed in \"History\" tab - #5873",
+      "[Added] Show progress bar in taskbar during pushing and pulling changes - #8433. Thanks @ChamodyaDias!",
+      "[Added] Add support for JetBrains Rider as external editor - #9368 #9365. Thanks @wengfatt and @say25!",
+      "[Added] Add support for PyCharm as external editor on macOS - #9401. Thanks @jakubroztocil!",
+      "[Added] Add support for Android Studio as external editor on macOS - #9332. Thanks @guidezpl!",
+      "[Added] Add support for Notepad++ as external editor on Windows - #9235. Thanks @say25!",
+      "[Fixed] Use correct install locaition of Typora to fix Typora detect issue - #9417. Thanks @TerryChan!",
+      "[Fixed] Display pull requests from upstream repository in pull request list - #6383",
+      "[Fixed] Restored developer tools accelerator on Windows - #9430. Thanks @jfgordon2!",
+      "[Fixed] Use upstream repository in commit messages links to GitHub - #9334",
```

Comment:
```suggestion
      "[Fixed] Use upstream repository in commit message links to GitHub - #9334",
```

Reply:
should this be an improved? no strong opinion but worth asking

Reply:
I think it's ok as fixed because currently the user gets a 404.